pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -214,3 +214,21 @@ func (pas *PodAutoscalerStatus) inStatusFor(status corev1.ConditionStatus, now t
 func (pas *PodAutoscalerStatus) duck() *duckv1beta1.Status {
 	return (*duckv1beta1.Status)(&pas.Status)
 }
+
+// GetDesiredScale returns the desired scale if ever set, or -1.
+func (pas *PodAutoscalerStatus) GetDesiredScale() int32 {
```

Comment:
These seem to be unused?

Reply:
Hopefully they will, since Jon asked me to add them :)